docs/source/flow_setup.rst

```diff
@@ -198,8 +203,11 @@ to activate the `flow` env. Type:
     source activate flow
     python examples/simulate.py ring --aimsun
 
-*Troubleshootig for Ubuntu users with Aimsun 8.4*: when you run the above example, you may get a subprocess.Popen error ``OSError: [Errno 8] Exec format error:``.  
-To fix this, go to the `Aimsun Next` main directory, open the `Aimsun_Next` binary with a text editor and add the shebang to the first line of the script ``#!/bin/sh``.
+*Troubleshootig for Ubuntu users with Aimsun 8.4*: when you run the above
```

Comment:
Troubleshooting

Reply:
I was just moving everything within 80 characters so it's easier to read